rotkehlchen/tests/unit/test_defi_oracles.py

```diff
@@ -102,3 +107,27 @@ def mocked_asset_getter(identifier, form_with_incomplete_data):  # pylint: disab
             inquirer_defi._uniswapv2.query_current_price(weth, A_USDC, False)
         with pytest.raises(DefiPoolError):
             inquirer_defi._uniswapv3.query_current_price(weth, A_USDC, False)
+
+
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
+def test_pool_with_no_liquidity(inquirer_defi: 'Inquirer'):
+    """
+    Test that a pool with no liquidity on range is skipped when using uni-v3 oracle
+    """
+    old_steram = EvmToken('eip155:1/erc20:0x0Cf0Ee63788A0849fE5297F3407f701E122cC023')
```

Comment:
what is a steram?

Reply:
the token is `stream data` and of course I had to typo it. I will write it in my red notebook of failures and fix it in another PR if it is the only thing (I ran test locally so I guess they will be fine)

Reply:
ofc